packages/reporter/src/errors/error-stack.tsx

```diff
@@ -0,0 +1,74 @@
+import _ from 'lodash'
+import { observer } from 'mobx-react'
+import React, { ReactElement } from 'react'
+
+import ErrorFilePath from './error-file-path'
+import Err from './err-model'
+
+const cypressLineRegex = /(cypress:\/\/|cypress_runner\.js)/
```

Comment:
i don't believe this is sufficient for production builds of cypress.


The `cypress://` protocol is only in development.

Reply:
wait nevermind, it has `cypress_runner.js` in there

Reply:
That's what the `cypress_runner.js` part is for.